backend/models/User.js

```diff
@@ -15,14 +15,12 @@ var UserSchema = new Schema({
             unique: true 
         }
     },
-    firstName: {
-        type: String,
-        required: true    
-    },
-    lastName: {
+    //lastName and firstName field change just name 
```

Comment:
@byklyci let's remove this comment to avoid any confusion.

Reply:
:+1: